app/src/main/java/com/eveningoutpost/dexdrip/Models/blueReader.java

```diff
@@ -78,26 +107,34 @@ public static boolean isblueReaderPacket(byte[] buffer) {
             return null;
         } else if (bufferstring.startsWith("HYBERNATE SUCCESS")) {
             Log.i (TAG, "blueReader notice that NFC is now really hibernated...");
-            if (counterHibernated > 0) {
+            if (counterHibernated > (int)0) {
```

Comment:
both counterHibernated and 0 are int. you can delete the (int) cast which is not needed.